packages/mobile/src/account/Education.tsx

```diff
@@ -68,6 +94,11 @@ export default class Education extends React.Component<Props, State> {
           currentStep: step,
           direction: ScrollDirection.previous,
         })
+      } else if (topic === EducationTopic.onboarding) {
+        ValoraAnalytics.track(OnboardingEvents.onboarding_education_scroll, {
+          currentStep: step,
+          direction: ScrollDirection.previous,
+        })
```

Comment:
This looks like it could be refactored to avoid repetition.
I didn't want to disrupt more code than necessary at this stage though.

Reply:
Yeah, we could just set a variable like `analyticsEvent` within the if statements then just call `ValoraAnalytics` once at the end. Don't think that would be an issue